zgw/verzoek/build.gradle

```diff
@@ -22,11 +22,18 @@ dockerCompose {
 }
 
 dependencies {
+    implementation project(":contract")
+    implementation project(":document")
+    implementation project(":process-document")
     implementation project(":plugin")
     implementation project(":plugin-authentication:notificaties-api-authentication")
+    implementation project(":openzaak")
```

Comment:
We should avoid using the :openzaak module. Since that that module contains an old openzaak-controller that we're trying to replace with the new zakenapi-plugin

Reply:
fixed